raiden-network/light-client
# Added UDC view

**Thank you for submitting this pull request :)**

Resolves #1489 

**Short description**


**Definition of Done**


**Steps to manually test the change (dApp)**

1. Serve dApp
2. Click identicon
3. UDC menu item should not be shown
4. Go back and connect
5. Click identicon
6. UDC icon should be shown
7. UDC view should display balance
8. Should display hint if balance is too low
9. Should allow minting and depositing
